src/javascript/DesignSystem/PickerDialog/PickerDialog.jsx

```diff
@@ -140,11 +151,15 @@ const PickerDialogCmp = ({
                     </div>
                     <Button
                         data-sel-picker-dialog-action="done"
-                        disabled={!selectedItem}
+                        disabled={!(
+                            // This following assertion is when the button is enabled (see unit test)
+                            (selectedItem && selectedItem.length !== 0) ||
+                            (initialSelectedItem && selectedItem === undefined)
```

Comment:
is it possible to extract the two parts of this condition to 2 distinct const to more readability ?